data-raw/crosswalk-zoopComb-zoopEnvComb-startDates.R

```diff
@@ -14,9 +14,9 @@ crosswalk<-readr::read_csv(file.path("data-raw", "crosswalk.csv"),
                                                Intro="d", EMPstart="d", EMPend="d",
                                                FMWTstart="d", FMWTend="d", twentymmstart="d",
                                                twentymmend="d", twentymmstart2="d", DOPstart="d",
-                                               DOPend="d"))%>%
-  dplyr::mutate_at(dplyr::vars(c("EMPstart", "EMPend", "Intro", "FMWTstart", "FMWTend", "twentymmstart", "twentymmend", "twentymmstart2", "DOPstart", "DOPend")), ~readr::parse_date(as.character(.), format="%Y"))%>%
-  dplyr::mutate_at(dplyr::vars(c("EMPstart", "FMWTstart", "twentymmstart", "twentymmstart2", "EMPend", "FMWTend", "twentymmend", "DOPstart", "DOPend")), ~tidyr::replace_na(., lubridate::ymd("2500-01-01")))%>% #Change any NAs for starts or ends to 2500 (i.e. never started or ended, super far in the future)
+                                               DOPend="d", FRPstart = "d"))%>%
```

Comment:
If start dates are added for FRP, we also need a column for FRP end dates, even if it's blank. FRP also needs to be added to the taxonomic consistency over time code, which is currently only applied to "EMP", "STN", "FMWT", "twentymm", "DOP". This would need to be changed in line 410 of the current Zoopsynther function.

Reply:
I think i did it right, you might want to double-check my work.